README.md

```diff
@@ -95,6 +115,27 @@ What are the differences between calling and referencing a function? Please prov
 
 ```text
 
+calling a function or invoking it excutes the function i.e
+function myNum (x){
+  console.log(x);
+}
+
+myNum(8);
+
+
+referencing a function is similar to calling, it gets passed as a parameter in another function i.e
+
+function myNum(x){
+  x = prompt("number?")
+}
+
+function addNum(p,y){
+ return p+y;
+}
+
+addNum(myNum(),7);
```

Comment:
the idea is to pass the function without the parens, reference without calling it.